### ISSUE TITLE
Fails to build on nixos

### ISSUE THREAD

**Issue by AusCyberman** [eraserhd/parinfer-rust#104]
cargosha256 is incorrect, which must be changed and also on build with nix-build and cargo build --release.

the error is

```
 --- stderr
  ./include/emacs-module.h:23:10: fatal error: 'stdint.h' file not found
  ./include/emacs-module.h:23:10: fatal error: 'stdint.h' file not found, err: true
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', /home/auscyber/.cargo/registry/src/github.com-1ecc6299db9ec823/emacs_module-0.16.2/build.rs:15:10
  stack backtrace:
     0: rust_begin_unwind
     1: core::panicking::panic_fmt
     2: core::result::unwrap_failed
     3: core::result::Result<T,E>::unwrap
     4: build_script_build::main
     5: core::ops::function::FnOnce::call_once
  note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by ranfdev** [comment]
NixOS user here.

- The 'stdint.h' file is not found. Partial fix: if you don't use emacs, don't compile the emacs dependencies at all. See pull request #103
- The cargosha256 is wrong. Fixed here #105

**Comment by eraserhd** [comment]
This seems to be fixed.  Thanks @ranfdev!

CI seems hopelessly broken though :(
